cinema/views.py

```diff
@@ -56,3 +91,56 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
+        date = self.request.query_params.get("date")
+        movie_id = self.request.query_params.get("movie")
+
+        if date:
+            year, month, day = date.split("-")
+            queryset = queryset.filter(
+                show_time__year=year,
+                show_time__month=month,
+                show_time__day=day
+            )
+        if movie_id:
+            queryset = queryset.filter(movie_id=int(movie_id))
```

Comment:
Also not a mistake, but a user may want to filter movie sessions by more than 1 movies. Methods from Movie views may come in handy.